Formula/jj.rb

```diff
@@ -26,6 +26,7 @@ class Jj < Formula
 
   def install
     system "cargo", "install", "--no-default-features", "--bin", "jj", *std_cargo_args
+    generate_completions_from_executable(bin/"jj", "debug", "completion", shell_parameter_format: "--")
```

Comment:
This should `:flag` rather than `--`? So that it would be translated into `jj debug completion --(bash|zsh|fish)`?
```suggestion
    generate_completions_from_executable(bin/"jj", "debug", "completion", shell_parameter_format: :flag)
```

Reply:
Yes, I test it on my machine. It's works. Thanks